Shubham_Sharma/Profile-bootstrap/Profile.html

```diff
@@ -0,0 +1,196 @@
+                    <h3 class="card-title">Xth</h3>
+                    <p class="card-text">Sarvada Modern Secondary School</p>
+                    <p class="card-text"><b>CBSE</b></p>
+                    <p class="card-text"><b>8.8 CGPA</b></p>
+                    <a href="http://smsschool.net/" class="btn btn-primary">About School</a>
+                </div>
+          </div>
+        </div>
+        <div class="col-sm-5 mb-3">
+            <div class="card bg-pic">
+                <div class="card-block text-center py-3">
+                    <h3 class="card-title">XIIth</h3>
+                    <p class="card-text"><b>Rukmani Devi Jaipuria Public School</b></p>
+                    <p class="card-text"><b>CBSE</b></p>
+                    <p class="card-text"><b>86.8 %</b></p>
+                    <a href="http://rdjps.co.in/" class="btn btn-primary">About School</a>
+                </div>
+          </div>
+        </div>
+        <div class="col-sm-1">
+
+        </div>
+    </div>
+
+    <div class="row mb-5">
+        <div class="col-sm-3">
+
+        </div>
+        <div class="col-sm-6">
+            <div class="card bg-pic">
+                <div class="card-block text-center py-3">
+                    <h3 class="card-title">B.Tech</h3>
+                    <p class="card-text"><b>Sharda University</b></p>
+                    <p class="card-text"><b>Computer Science</b></p>
+                    <p class="card-text"><b>8.46 CGPA</b></p>
+                    <a href="https://www.sharda.ac.in/" class="btn btn-primary">About Institute</a>
+                </div>
+            </div>           
+        </div>
+        <div class="col-sm-3">
+
+        </div>    
+    </div>
+
+    <hr>
+
+    <!--Technical Knowledge-->
+    <h1 class="text-center my-3 display-3" id="techKnowledge">Technical Knowledge</h1>
+    <div class="row">
+        <div class="col-sm-1">
+
+        </div>
+        <div class="col-sm-5 mb-3">
+            <div class="card bg-pic">
+                <div class="card-block text-center py-3">
+                    <h3 class="card-title">HTML</h3>
+                    <p class="card-text">Intermediate</p>
+                    <footer class="blockquote-footer"><b>On my way to <cite title="Source Title">Become Professional</cite></b></footer>
+                </div>
+          </div>
+        </div>
+        <div class="col-sm-5 mb-3">
+            <div class="card bg-pic">
+                <div class="card-block text-center py-3">
+                    <h3 class="card-title">CSS</h3>
+                    <p class="card-text"><b>Intermediate</b></p>
+                    <footer class="blockquote-footer"><b>On my way to <cite title="Source Title">Become Professional</cite></b></footer>
+                </div>
+          </div>
+        </div>
+        <div class="col-sm-1">
+
+        </div>
+    </div>
+    <div class="row mb-5">
+        <div class="col-sm-3">
```

Comment:
why empty div ?

Reply:
I am using bootstrap grid, so to center the elements, I am using this empty div and setting columns.